### DISCUSSION TITLE
Grpc

### DISCUSSION THREAD

**Discussion by YKWebapi** [abpframework/abp#7569] [General]
ABP的跨模块支持grpc嘛

**Comment by maliming** [comment]
Not for now.

**Comment by YKWebapi** [comment]
When is there a plan for ABP to support GRPC across modules, as.NET 5 already supports it




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "abpframework/abp"                                                                                    <notifications@github.com&gt;;
发送时间:&nbsp;2021年2月2日(星期二) 下午5:21
收件人:&nbsp;"abpframework/abp"<abp@noreply.github.com&gt;;
抄送:&nbsp;"421868450"<421868450@qq.com&gt;;"Author"<author@noreply.github.com&gt;;
主题:&nbsp;Re: [abpframework/abp] Grpc (#7569)





 
Not for now.
